plugins/text-freetype2/data/text_default.effect

```diff
@@ -18,7 +19,7 @@ VertInOut VSDefault(VertInOut vert_in)
 	VertInOut vert_out;
 	vert_out.pos = mul(float4(vert_in.pos.xyz, 1.0), ViewProj);
 	vert_out.uv  = vert_in.uv;
-	vert_out.col = vert_in.col;
+	vert_out.col = use_color ? vert_in.col : float4(1.0, 0.0, 0.0, 0.0);
```

Comment:
I also see the outline and drop-shadow is not displayed on Linux (Fedora 38, X11). Probably, this will fix it.
```suggestion
	vert_out.col = use_color ? vert_in.col : float4(0.0, 0.0, 0.0, 1.0);
```

Reply:
Done, thanks.